src/components/ConfirmModal.tsx

```diff
@@ -2,10 +2,10 @@ import { FC, useContext } from "react";
 import { createPortal } from "react-dom";
 import { useTranslation } from "react-i18next";
 import { useNavigate } from "react-router";
-import ModalDialog from "../../../container/ModalDialog/ModalDialog";
-import { AppContext } from "../../../store/app-context";
-import { instance } from "../../../util/interceptor";
-import { MODAL_ROOT } from "../../../util/util";
+import ModalDialog from "../layouts/ModalDialog";
+import { AppContext } from "../context/app-context";
+import { instance } from "../util/interceptor";
+import { MODAL_ROOT } from "../util/util";
```

Comment:
Later `util.ts` could be renamed to `index.ts` and import could just be `from "../util";`

Reply:
Thank you! Done